tuna/miopen/db/__init__.py

```diff
@@ -0,0 +1,25 @@
+###############################################################################
```

Comment:
Why add a license to a blank file?

Reply:
To avoid the import fail error - in submodules.

Reply:
Leave the file, remove the license.